app/main.py

```diff
@@ -2,0 +1,16 @@
+def copy_file(command: str) -> None:
+    splitted = command.split(" ")
+    if len(splitted) != 3:
+        return "Invalid input"
+    cm = splitted[0]
+    initial_file_name = splitted[1]
+    new_file_name = splitted[2]
+    if initial_file_name == new_file_name:
+        return None
```

Comment:
The function should return a boolean value to indicate success or failure. Consider returning `False` here to indicate that the operation was not successful.